docs/OnTrack/Voice Verification/Handbook for Voice Verification.md

```diff
@@ -0,0 +1,85 @@
+# Voice Verification Handbook
+
+## Table of Contents:
+
+---
+
+- OnTrack Local Setup
+
+- Voice Verification Setup – ToDo
+
+- Ruby and Rails Tutorial Link
+
+- [Install Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+- [Install Git](https://git-scm.com/downloads)
+
+  - For Windows 10, turn on Hyper V (Windows 10 Enterprise, Pro, or Education only).
+
+---
+
+**TODO: `FIX THESE LINKS`**
+
```

Comment:
Please Fix these links. Make sure that the documents are accessible, either by adding them to the PR, or through other means.